README.md

```diff
@@ -1,112 +1,102 @@
-# Twitter API V2
+# Twitter API v2
 
-Twitter api V1 and V2 client for node
+Strongly typed, full-featured, right protected, versatile yet powerful Twitter API v1.1 and v2 client for Node.js.
 
-## How to use
+## Why?
 
-```typescript
-import TwitterApi, { TwitterErrors } from 'twitter-api-v2';
+- The main libraries (twit/twitter) were not updated in a while
```

Comment:
I know you kept what I wrote, maybe we can rephrase these (only 1 small dependency etc...)

Or maybe we can remove it and replace it with ex. a comparison with other libraries. Let's rediscuss it later offline :)